### PR TITLE
Update workshop scripts and instructions

### PR DESCRIPTION
**Pull request by emizzle**
The scripts work on *nix-like environments. They should probably be improved to work on windows-native environments, however the existing windows scripts are still there.

These have been tested on both macos and linux.

The workshop instructions have been updated to reflect the new scripts with additional information and emojis (ha)

### CODE REVIEW

File: scripts/windows/download.bat [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+set "OS=windows"
+call :get_arch ARCH
+set "ARCHIVE_EXT=.zip"
+set "EXE_EXT=.exe"
+set "VERSION=v0.1.0"
+set "BASE_URL=https://github.com/codex-storage/nim-codex/releases/download/%VERSION%"
+set "EXTRACT_DIR=.\"
+set "BINARY_NAMES=codex"
+
+:: Download, verify, and extract each binary
```
Comment (jessiebroke) [inline scripts/windows/download.bat:16]: Does not properly verify checksum of binary and extract from .zip file.

Testing it on W11 machine in Powershell (not UCRT or MSYS) and ran into checksum issue in download.bat.

image

Failed on executing Codex binary (didn't extract zip). 

image

**Review by jessiebroke — CHANGES_REQUESTED** [review]
Works in Ubuntu 22.04 x64. Does not work in W11 x64. See comments and screenshots.

File: WORKSHOPSETUP.md [hunk 2]
```diff
@@ -1,51 +1,79 @@
...
+# LOCAL IP: 192.168.50.45
+# INF 2024-06-25 16:32:25.410+10:00 Creating a private key and saving it       tid=25484256
+# INF 2024-06-25 16:32:25.414+10:00 Discovery SPR initialized                  topics="discv5"
+# ...
+```
 
-Run `run_client` script.
+> #### 📢 **Don't lose your generated address**<br>
+>Check that your `LOCAL IP` in the output is correct, and if not, re-run the script with
+the LOCALfIP environment variable:
```
[the remark this answers is not in the document]
Comment (emizzle) [reply WORKSHOPSETUP.md:56]: the env variable is called `LOCALIP`

File: scripts/windows/download.bat (hunk 1 above)
Comment (emizzle) [reply scripts/windows/download.bat:16]: These scripts were auto-ported by Claude to native windows and are not fully functional yet. The first error with shasum checking is a case-senstive check which should be easy to fix. We haven't had a chance to get to the rest. Any and all help on Windows is appreciated. For now, we're gonna leave them as is and work with the assumption that everyone on windows has a *nix-like environment on their machine, and we can update these scripts in the coming days

**Pull request merged**